src/main/webapp/app/shared/pipes/filterfeedbacks.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { PipeTransform, Pipe } from '@angular/core';
+import { Feedback } from 'app/entities/feedback';
+
+@Pipe({ name: 'filterfeedbacks' })
+export class FilterFeedbacksPipe implements PipeTransform {
```

Comment:
This pipe needs a better name and/or documentation.
As far as I see, it is a RemovePositiveFeedback or FilterNegativeFeedback, isn't it?

Reply:
RemovePositiveAutomaticFeedback should be the best name then, I guess.